<ui version="4.0" >
 <class>MyForm</class>
 <widget class="QWidget" name="MyForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>DesignedWidget</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLCDNumber" name="m_lcd" />
     </item>
     <item>
      <widget class="QSlider" name="m_sld" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="m_cmdReset" >
       <property name="text" >
        <string>&amp;Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_cmdQuit" >
       <property name="text" >
        <string>&amp;Quit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_sld</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>m_lcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>177</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel" >
     <x>177</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cmdQuit</sender>
   <signal>clicked()</signal>
   <receiver>MyForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>346</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel" >
     <x>395</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
